src/bin/api-gen/message.rs

```diff
@@ -136,7 +136,7 @@ impl VppJsApiMessage {
                         VppJsApiFieldSize::Variable(t) => {
                             code.push_str(&format!("VariableSizeArray<{}>, \n", get_type(&self.fields[x].ctype)))
                         }
-                        _ => code.push_str(&format!("{},\n", get_type(&self.fields[x].ctype))),
+                        //_ => code.push_str(&format!("{},\n", get_type(&self.fields[x].ctype))),
```

Comment:
why is this code specific to single file code generation ?

Reply:
This was an unreachable pattern, Because we are already matching for all the variants, It's too not have the warning during execution. It's not relevant to the PR, I didn't want to commit a single line change, That's the reason why this is here.